diff --git a/tinyxml2.cpp b/tinyxml2.cpp
index 3043ab3..0fd2d76 100755
--- a/tinyxml2.cpp
+++ b/tinyxml2.cpp
@@ -2024,7 +2024,7 @@ void XMLDocument::Clear()
 		DeleteNode(_unlinked[0]);	// Will remove from _unlinked as part of delete.
 	}
 
-#ifdef DEBUG
+#ifdef TINYXML2_DEBUG
     const bool hadError = Error();
 #endif
     ClearError();
@@ -2039,7 +2039,7 @@ void XMLDocument::Clear()
     _attributePool.Trace( "attribute" );
 #endif
     
-#ifdef DEBUG
+#ifdef TINYXML2_DEBUG
     if ( !hadError ) {
         TIXMLASSERT( _elementPool.CurrentAllocs()   == _elementPool.Untracked() );
         TIXMLASSERT( _attributePool.CurrentAllocs() == _attributePool.Untracked() );
diff --git a/tinyxml2.h b/tinyxml2.h
index e6a0ced..dfd0fa1 100755
--- a/tinyxml2.h
+++ b/tinyxml2.h
@@ -54,8 +54,8 @@ distribution.
 */
 
 #if defined( _DEBUG ) || defined (__DEBUG__)
-#   ifndef DEBUG
-#       define DEBUG
+#   ifndef TINYXML2_DEBUG
+#       define TINYXML2_DEBUG
 #   endif
 #endif
 
@@ -79,7 +79,7 @@ distribution.
 #endif
 
 
-#if defined(DEBUG)
+#if defined(TINYXML2_DEBUG)
 #   if defined(_MSC_VER)
 #       // "(void)0," is for suppressing C4127 warning in "assert(false)", "assert(true)" and the like
 #       define TIXMLASSERT( x )           if ( !((void)0,(x))) { __debugbreak(); }
@@ -390,7 +390,7 @@ public:
         }
         --_currentAllocs;
         Item* item = static_cast<Item*>( mem );
-#ifdef DEBUG
+#ifdef TINYXML2_DEBUG
         memset( item, 0xfe, sizeof( *item ) );
 #endif
         item->next = _root;
